### PR TITLE
CDAP-4767: Transaction service cannot be discovered on a secure cluster

### PR DESCRIPTION
**Pull request by jwang47**

### CODE REVIEW

File: cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java [hunk 1]
```diff
@@ -632,6 +632,8 @@
     public static final String ENABLED = "security.enabled";
     /** Enables Kerberos authentication. */
     public static final String KERBEROS_ENABLED = "kerberos.auth.enabled";
+    /** Kerberos keytab relogin interval. */
+    public static final String KERBEROS_KEYTAB_RELOGIN_INTERVAL = "kerberos.auth.relogin.interval.s";
```
Comment (chtyim) [inline cdap-common/src/main/java/co/cask/cdap/common/conf/Constants.java]: Better call it `.seconds`.

**Comment by chtyim** [comment]
The changes LGTM. We should verify it on a cluster before merging it.

**Comment by jwang47** [comment]
https://issues.cask.co/browse/CDAP-4767
http://builds.cask.co/browse/CDAP-RBT641

**Comment by jwang47** [comment]
CDAP starts up fine in a new cluster without issues as expected. Ideally, I'd have this cluster running for at least 8 days, but we likely won't have enough time. Going to leave the cluster running for as long as possible.

**Pull request merged**